metomi/rose/scripts/rose-mpi-launch

```diff
@@ -128,7 +128,7 @@ while (($# > 0)); do
     --help)
         rose_help
         exit
-        :;;
+        ;;
```

Comment:
Gotta do this for the other branches too.

Reply:
Only the other branch with an exit in it.